exodus_core/analysis/static_analysis.py

```diff
@@ -344,16 +344,16 @@ def _get_icon_from_gplay(self, handle, path):
         ICON_CLASS = 'T75of ujDFqe'
 
         address = 'https://play.google.com/store/apps/details?id=%s' % handle
-        gplay_page_content = urllib.request.urlopen(address).read()
+        gplay_page_content = requests.get(address).text
         soup = BeautifulSoup(gplay_page_content, 'html.parser')
         icon_images = soup.find_all('img', {'class': ICON_CLASS})
         if len(icon_images) > 0:
             icon_url = '%s' % icon_images[0]['src']
             if not icon_url.startswith('http'):
                 icon_url = 'https:%s' % icon_url
-            f = urllib.request.urlopen(icon_url)
+            f = requests.get(icon_url)
             with open(path, mode='wb') as fp:
-                fp.write(f.read())
+                fp.write(f.content)
```

Comment:
why `f.content` and not `f.text` ?

Reply:
I did not choose poorly. 

Attribute .text has the value as strings of the response. From another hand, .content works as binary. It is necessary to change this attribute to be understood as argument of the function write() which takes a binary object as content.